api/v2/v2.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package v2
+
+import (
+	"github.com/decred/dcrtime/merkle"
+)
+
+const (
+	// StatusRoute defines the API route for retrieving
+	// the server status.
+	StatusRoute = "/v2/status/"
+
+	// TimestampRoute defines the API route for submitting
+	// a single string digest.
+	TimestampRoute = "/v2/timestamp/" // Single digest timestamping
+
+	// TimestampsRoute defines the API route for submitting
+	// a batch of timestamps or digests.
+	TimestampsRoute = "/v2/timestamps/" // Multi digest timestamping
+
+	// VerifyRoute defines the API route for both timestamp
+	// and digest verification.
+	VerifyRoute = "/v2/verify/" // Multi verify digests
+
+	// ResultOK indicates the operation completed successfully.
+	ResultOK = 0
```

Comment:
You typically want the 0 value to be invalid, so in this case `ResultInvalid = 0`.  It looks like you did it this way to mimic the v1 behavior.  Also, shouldn't these be a specific `ResultT` type instead of a generic int?

@marcopeereboom what do you think about this?

Reply:
I agree that we should create a specific type for those errors, `ResultT`. Will commit this in a moment, and wait on marco's take about either we should change the values or not.

Reply:
Hey mate @lukebp changed the result code 0 to be invalid :+1: